static/src/js/containers/MapContainer/MapEvents.jsx

```diff
@@ -10,9 +15,44 @@ const MapEvents = (props) => {
     onMetricsMap
   } = props
 
+  const [showTooltip, setShowTooltip] = useState(false)
+  const [tooltipTarget, setTooltipTarget] = useState(null)
+
+  const handleMouseOver = useCallback((event) => {
+    setShowTooltip(true)
+    setTooltipTarget(event.currentTarget)
+  }, [])
+
+  const handleMouseOut = useCallback(() => {
+    setShowTooltip(false)
+    setTooltipTarget(null)
+  }, [])
+
   useLayoutEffect(() => {
```

Comment:
Since we've added some event listeners, we need to make sure to provide a cleanup callback function in a return statement at the end of the function. 

Should end up looking like:
```
return () => {
    control.removeEventListener('mouseover', handleMouseOver)
    control.removeEventListener('mouseout', handleMouseOut)
}
```